packages/radio-box-group/src/RadioBox.tsx

```diff
@@ -33,129 +33,92 @@ const inputStyles = css`
 interface StateForStyles {
   checked: boolean;
   disabled: boolean;
-  showFocus: boolean;
+  size: Size;
 }
 
-const getRadioDisplayStyles = ({
-  checked,
-  disabled,
-  showFocus,
-}: StateForStyles) => {
+const getInteractionRingStyles = ({ checked }: StateForStyles) => {
   const baseStyles = css`
-    transition: border 150ms ease-in-out;
-    box-sizing: content-box;
-    padding: 15px;
-    cursor: pointer;
-    font-size: 14px;
-    font-weight: bold;
-    text-align: center;
-    vertical-align: top;
-    overflow-wrap: break-word;
-    background-color: white;
-    border: 1px solid ${uiColors.gray.light1};
-    color: ${uiColors.gray.dark2};
-    border-radius: 3px;
-    z-index: 2;
-
-    ${radioBoxWrapper.selector}:hover & {
-      border-color: ${uiColors.gray.base};
-    }
+    width: 100%;
+    // Display behind border
+    z-index: -1;
   `;
 
-  if (disabled) {
+  if (checked) {
```

Comment:
Should we add `active` to the force states in the InteractionRing component? That way we don't have to manually change the zindex in all of the components consuming the interaction ring (since it's largely form elements, most do have an active state)

Reply:
Are you proposing that we make the interaction ring have a negative z-index for the active state? I think this wouldn't make a difference for most components since there's no overlap between the interaction ring and the surrounded element. `RadioBox` is kind of exceptional in that sense and I think I've heard that we have some interest updating the design of `RadioBox` at some point

Reply:
Kk reasonable enough, was just wondering if this was repeated logic around the consuming components that may as well be shared while we're building this out. But if this is more exceptional than I thought, then I'm fine to leave as-is